### DISCUSSION TITLE
Need help using MicroPython on EV3

### DISCUSSION THREAD

**Discussion by HelpMeP1s** [pybricks/support#1952] [Q&A]
Hi im new in micropython coding, can anyone please help me with these problems;
1. when i try to Run and Debug thees code:
```python
from pybricks.ev3devices import Motor, ColorSensor
from pybricks.parameters import Port, Direction
from pybricks.tools import  wait
from pybricks.robotics import DriveBase 
from pybricks.hubs import EV3Brick
```
i recive this problem
```console
"Starting: brickrun --directory="/home/robot/pid_2" "/home/robot/pid_2/main.py"
----------
Traceback (most recent call last):
  File "/home/robot/pid_2/main.py", line 7, in <module>
ImportError: no module named 'pybricks.hubs'
Exited with error code 1.
----------
```
2. when i try and use "The Control Class" for PID motor controll, my VS code script is not seeng it. Can anyone shere an example of using The Control Class?
3. Alsoe apriciate a lot if anyone can share their LINE FOLLOWER code for EV3 robot with 2 sensors
Thanks in advanse.

**Comment by dlech** [comment]
That is strange that the other imports work but that one doesn't. It would make more sense if all worked or none worked since they are all coming from the same package.

By chance are you using a SD card bigger than 32GB? That can cause files to mysteriously disappear.

**Reply by HelpMeP1s** [reply]
16Gb

**Comment by laurensvalk** [comment]
`pybricks.hubs` was introduced in 2.0. You may be using version 1.0, which is very old. Where did you download it?

See https://pybricks.com/ev3-micropython/startupgrade.html

**Reply by HelpMeP1s** [reply]
I checked, version is 2.0

**Reply by laurensvalk** [reply]
Which download link did you use?
